bin/update-helper.sh

```diff
@@ -63,23 +63,32 @@ function composer_update_outdated() {
       # Composer files:
       git add composer.json composer.lock
 
-      # Drupal scaffold files:
-      git add web
+      package_type=$(composer show $c | grep type | awk '{print $3}')
```

Comment:
To prevent capturing another line that contains the 'type' word:

```suggestion
      package_type=$(composer show $c | grep ^type | awk '{print $3}')
```